api/v1beta1/activemqartemis_types.go

```diff
@@ -570,6 +570,9 @@ type AcceptorType struct {
 	// Whether to let the acceptor to bind to all interfaces
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Bind To All Interfaces",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:booleanSwitch"}
 	BindToAllInterfaces *bool `json:"bindToAllInterfaces,omitempty"`
+	// Type of keystore being used; "JKS", "JCEKS", "PKCS12", etc. Default in broker is "JKS"
```

Comment:
in the comment, as this gets into the yaml

Reply:
I'll add PEMCFG to the yaml.

Reply:
actually, I would not add `PEM` or `PEMCFG` because if my understanding is correct this PR doesn't add a full support for those key store types. I f a user want to use the secrect generate by the cert/trust managers the KeyStoreType should be empty.

Reply:
@brusdev yeah that makes sense.